src/main/kotlin/no/nav/familie/ef/sak/beregning/barnetilsyn/BeregningBarnetilsynUtil.kt

```diff
@@ -94,7 +94,7 @@ object BeregningBarnetilsynUtil {
             kalkulerUtbetalingsbeløpFørFratrekkOgSatsjustering(periodeutgift, kontantstøtteBeløp)
 
         val maxSatsBeløp = when (brukIkkeVedtatteSatser) {
-            true -> ikkeVedtatteSatserForBarnetilsyn.hentSatsFor(antallBarn, årMåned).toBigDecimal()
+            true -> ikkeGjeldendeSatserForBarnetilsyn.hentSatsFor(antallBarn, årMåned).toBigDecimal()
```

Comment:
Synes det er litt skummelt å gjenbruke featuretoggle for dette, når den heter `brukIkkeVedtatteSatser`